client/dashboard/task-list/style.scss

```diff
@@ -53,3 +64,74 @@
 		position: relative;
 	}
 }
+
+.woocommerce-shipping-rates {
+	margin-bottom: $gap;
+}
+
+.woocommerce-shipping-rate {
+	display: flex;
+	padding-top: $gap-small;
+	padding-bottom: $gap-small;
+
+	.woocommerce-shipping-rate__main {
+		width: 100%;
+	}
+
+	.woocommerce-shipping-rate__icon {
+		padding-top: $gap;
+		margin-right: $gap-large;
+	}
+
+	.woocommerce-shipping-rate__name {
+		align-items: center;
+		display: flex;
+		padding-top: $gap;
+		font-size: 16px;
+		line-height: 22px;
+		color: #1a1a1a;
```

Comment:
`$muriel-gray-900` is `#1a1a1a`.

Reply:
Thanks!  Updated here and a few other spots as well.